ggircs-app/app/components/XmlDiff/RenderDiff.tsx

```diff
@@ -54,22 +54,19 @@ const RenderDiff: React.FC<Props> = ({ query }) => {
     query
   );
 
-  const shouldRenderDiff =
-    firstSideReport &&
-    secondSideReport &&
-    router.query.FirstSideRelayId &&
-    router.query.SecondSideRelayId;
-  if (!shouldRenderDiff) return null;
```

Comment:
This probably shouldn't have made it in in the first place. Can you re-enable the "rules of hooks" linter rule, so that we don't allow this in he future: https://github.com/bcgov/cas-ggircs/blob/b25e729f49a4968b2a14bb2124310be84081afeb/ggircs-app/app/.eslintrc.js#L16